configs/swarm/docker-compose.local.yml

```diff
@@ -22,7 +22,12 @@ services:
     init: true
     command: |-
       /bin/sh -c "
-      kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 --out-file $$GENESIS_SIGNED_FILE &&
+      kagami genesis sign /config/genesis.json \\
+          --public-key $$GENESIS_PUBLIC_KEY \\
+          --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 \\
```

Comment:
Why do we get public key from environment, but private key is always the same?

Reply:
Originally it was made this way because genesis private key is used only to sign genesis, and genesis public key is used also by `irohad`. But there are plans (https://github.com/hyperledger/iroha/pull/4719#discussion_r1638122530) to move genesis private key to env, so I moved genesis private key to env in this PR